Source/iOS/Extensions/UIView+Frame.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+public extension UIView {
+    
+    public var height: CGFloat {
```

Comment:
yes @onmyway133  i also got this idea from some else. Just standing on the [shoulders of giants](https://en.wikipedia.org/wiki/Standing_on_the_shoulders_of_giants)